### PR TITLE
Changed success page info for ADA cases

### PR DESCRIPTION
**Pull request by JamieHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-6527

### Change description ###
Added conditional logic to the confirmation page so that a unique message is displayed only for ADA cases.

**Found an content issue on the success page when testing [RIA-6495](https://tools.hmcts.net/jira/browse/RIA-6495), If it is Non ADA case then the Success page content should not change when we list the case, please check..**

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ListCaseConfirmationTest.java [hunk 1]
```diff
@@ -33,41 +37,62 @@ class ListCaseConfirmationTest {
 
     private ListCaseConfirmation listCaseConfirmation = new ListCaseConfirmation();
 
-    @Test
-    void should_return_confirmation() {
+    @ParameterizedTest
+    @EnumSource(value = YesOrNo.class, names = { "NO", "YES" })
+    void should_return_confirmation(YesOrNo yesOrNo) {
```
Comment (neha-aggarw) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ListCaseConfirmationTest.java:42]: can you please change the name of the parameter as 'isAda'?

**Review by neha-aggarw — CHANGES_REQUESTED** [review]

File: src/functionalTest/resources/scenarios/RIA-412-RIA-1571-list-case-confirmation.json [hunk 2]
```diff
@@ -7,15 +7,16 @@
       "eventId": "listCase",
       "state": "prepareForHearing",
       "caseData": {
-        "template": "minimal-appeal-submitted.json"
+        "template": "minimal-appeal-submitted.json",
+        "isAcceleratedDetainedAppeal": "No"
```
Comment (ThomasKKC) [inline src/functionalTest/resources/scenarios/RIA-412-RIA-1571-list-case-confirmation.json:11]: Can you add another json (copy of this) where isAcceleratedDetainedAppeal is "Yes"? and then return the correct expectation body

Comment (neha-aggarw) [reply src/functionalTest/resources/scenarios/RIA-412-RIA-1571-list-case-confirmation.json:11]: There is already one for ADA: minimal-ada-appeal-submitted.

Comment (neha-aggarw) [reply src/functionalTest/resources/scenarios/RIA-412-RIA-1571-list-case-confirmation.json:11]: Also looking at it, this test is not ada specific. So this should stay the same with isAcceleratedDetainedAppeal = No to the template for default normal flow.
but we should have another test for ADA flow.

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ListCaseConfirmationTest.java (hunk 1 above)
Comment (JamieHMCTS) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ListCaseConfirmationTest.java]: I have changed the name of parameter yesOrNo to isAda

**Review by ThomasKKC — APPROVED** [review]

**Review by neha-aggarw — APPROVED** [review]
LGTM - the functional test will be added in a separate PR.

**Pull request merged**